test/main.spec.js

```diff
@@ -74,8 +73,8 @@ describe('authorizeAccessToken()', () => {
       header: {typ: 'JWT', alg: 'HS256', kid: '194B72684'},
       payload: {
         iss: 'https://issuer.example.com',
-        iat: 1613655020,
-        exp: 1645191020,
+        iat: Math.floor(Date.now() / 1000),
+        exp: Math.floor(Date.now() / 1000),
```

Comment:
These values were different and static before, but now are (almost) the same.  Is that ok for what the tests are looking for?  Are these values supposed to be exactly the same?  If so, the would want to just call `now()` once and reuse the value.